cmd/utils/flags.go

```diff
@@ -545,7 +545,7 @@ func MakeEtherbase(accman *accounts.Manager, ctx *cli.Context) common.Address {
 	}
 	// If no etherbase was specified and no accounts are known, bail out
 	accounts, _ := accman.Accounts()
-	if etherbase == "" && len(accounts) == 0 {
+	if (etherbase == "" || etherbase == "0") && len(accounts) == 0 {
```

Comment:
Shouldn't this rather simply be `etherbase == "" || len(accounts) == 0`?

I.e. if I haven't specified any, or there aren't any to be found?

Edit: Moved the comment into the PR... for some reason it went into a dangling commit previously.